src/blob/BlobServer.ts

```diff
@@ -57,10 +58,16 @@ export default class BlobServer extends ServerBase implements ICleaner {
     const port = configuration.port;
 
     // We can create a HTTP server or a HTTPS server here
-
-    const httpServer = !configuration.hasCert()
-      ? http.createServer()
-      : https.createServer(configuration.getCert());
+    let httpServer;
+    let certOption = configuration.hasCert();
+    switch (certOption) {
+      case CertOptions.MkCert:
```

Comment:
Why do you need to distinguish between MkCert and DevCert?

I don't want the HTTPS implementation to have a notion of MkCert or DevCert.  So it's either HTTPS or HTTP.  If you need to distinguish between PEM or PFX, then use PEM or PFX.